packages/edgedb-rag-sdk/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@edgedb/rag-sdk",
```

Comment:
This name feels a bit too vague: What about `@edgedb/ai-vercel-provider`?

Reply:
hmm we can include vercel in the name, none of the [community providers](https://sdk.vercel.ai/providers/community-providers) do that tho, maybe just ai-provider is better and having in the package.json description, in the readme and in the docs the explanation that it is for vercel is enough... but also, we can include it in the name, I just don't think it's so crucial

Reply:
Portkey's provider is called `@portkey-ai/vercel-provider`, but yeah, most of them are called `something-ai-provider`, so `@edgedb/ai-provider` seems _fine_ but also a little weird considering we have `@edgedb/ai`. Doesn't seem clear that this is a provider for the vercel `ai` package, but maybe that's really just a problem with vercel calling the package `ai` instead of `@vercel/ai` 😅

Reply:
oh yes, I think their package should have included vercel, u go on npm and just install ai, it is weird... regarding edgedb, maybe we should ask more people for their thoughts

Reply:
hmm also I dont know will we write more providers in the future, so actually I believe is ok to call it  @edgedb/ai-vercel-provider or @edgedb/vercel-ai-provider, but I am not sure which one of these two is better

Reply:
> But now I have a problem with how do we call our interfaces and functions, atm we use: EdgeDBRagProvider, createEdgeDBRag, default instance is edgedbRag... Other LLMs call it mistralAI for example but they are AI , EdgeDB is not. How it should look on the user side?

Good question. We have both a language model (enhanced with RAG) and an embeddings model, right? I think mostly the pattern seems to be (looking at the OpenAI docs):

```ts
import { createEdgeDB } from "@edgedb/vercel-ai-provider";

const edgedb = createEdgeDB();

const languageModel = edgedb("some-language-model-name");

const embeddingsModel = edgedb.textEmbeddingModel("some-embeddings-model-name");
```

I don't think the distinction between this being a RAG-enhanced language model is super important at the naming level, but we should certainly spend plenty of time explaining it in the documentation and README. This isn't something I feel very strongly about so if anyone else feels like it's confusing that we don't mention RAG in the naming, I'm fine with it. `edgedDBRag.textEmbeddingModel` feels a little strange to me, but not a big deal.